src/components/button/index.md.njk

```diff
@@ -50,6 +50,28 @@ Only use disabled buttons if research shows it makes the user interface easier t
 
 {{ example({group: "components", item: "button", example: "disabled", html: true, nunjucks: true, open: false}) }}
 
+### Stop users from accidentally sending information more than once
+
+Sometimes, users double click buttons because they:
+
+- have used operating systems where they have to double click items to make them work
+- are experiencing a slow connection which means they are not given feedback on their action quickly enough
+- have motor impairments such as hand tremors which cause them to click the button involuntarily 
+
+In some cases, this can mean their information is sent twice. 
+
+For example, the GOV.UK Notify team discovered that a number of users were receiving invitations twice, because the person sending them was double clicking the “send” button. 
+
+If you are working in production and research shows that users are frequently sending information twice, you can configure the button to ignore the second click. 
+
+To do this, set the `data-prevent-double-click` attibute to `true`. You can do this directly in the HTML or, if you’re using Nunjucks, you can use the Nunjucks macro as shown in this example.
+
+{{ example({group: "components", item: "button", example: "preventDoubleClick", html: true, nunjucks: true, open: false}) }}
+
+This feature will prevent double clicks for users that have JavaScript enabled, however you should also think about the issue server-side to protect against attacks.
+
+In the case of slow connections, aim to give the user information about what’s happening, for example, by showing a loading spinner or a modal, before using `preventDoubleClick`. 
```

Comment:
We no longer reference the `preventDoubleClick` macro option above, so referencing it again here could cause confusion.

Reply:
Fixed!